core/plugins/org.polarsys.capella.core.platform.sirius.sirius.validation/src/org/polarsys/capella/core/platform/sirius/sirius/validation/ddiagram/DeleteInValidHyperLinkInDescription.java

```diff
@@ -47,13 +48,34 @@ public DeleteInValidHyperLinkInDescription() {
     _currentElementDescription = null;
   }
 
-  public boolean updateDescription(List<EObject> modelElements) {
+  private String getDescription(EObject object) {
```

Comment:
Code duplication. Same remark, look for existing helper, if not exist, contribute a new one.